experimental/dds/tree2/src/test/shared-tree/sharedTree.spec.ts

```diff
@@ -1227,22 +1224,15 @@ describe("SharedTree", () => {
 			validateTree(tree2, [expectedState]);
 		});
 
-		it.skip("can rebase cross-field move over delete", () => {
+		it("can rebase value change of node over cross-field move of that node", () => {
```

Comment:
Does it still make sense to keep that test now that we have the test in editing.spec.ts?